en/components/date_time_editor.md

```diff
@@ -84,7 +84,7 @@ IgxDateTimeEditor supports different display and input formats.
 
 For display, it uses Angular's [DatePipe](https://angular.io/api/common/DatePipe) which allows it to support predefined format options, such as `shortDate` and `longDate`. It can also accept a constructed format string using characters supported by the DatePipe, e.g. `EE/MM/yyyy`. Notice that formats like shortDate, longDate, etc., can be used as `displayFormat` only.
 
-To set specific input format, pass it as a string to the IgxDateTimeEditor directive. This will set both the expected user input format and [mask](mask.md) for the editor. 
+To set specific input format, pass it as a string to the IgxDateTimeEditor directive. This will set both the expected user input format and [mask](mask.md) for the editor.
```

Comment:
```suggestion
To set a specific input format, pass it as a string to the IgxDateTimeEditor directive. This will set both the expected user input format and the [mask](mask.md) for the editor.
```